stat11_backend/stat11/views/match.py

```diff
@@ -5,23 +5,36 @@
 from rest_framework.filters import OrderingFilter
 from stat11.models import Match
 from stat11.serializers import MatchSerializer, MatchNestedSerializer
-from stat11.utils import get_match_team_data
+from stat11.utils import get_match_team_data, get_match_mvp_data , get_match_team_list_data
 
 class MatchModelViewSet(viewsets.ModelViewSet):
     queryset = Match.objects.all()
     filter_backends = [DjangoFilterBackend, OrderingFilter]
     filterset_fields = ['tournament__id', 'date', 'time', 'overs_no']
     ordering_fields = ['date', 'time']
-
     def get_serializer_class(self):
         if self.action == 'list' or self.action == 'retrieve':
             return MatchNestedSerializer
         return MatchSerializer
     
     @action(detail=False, methods=['get'])
     def all_match_and_teams(self, request):
-
         all_match_list = self.get_queryset()
         first_match = all_match_list[0]
         res = get_match_team_data(first_match.id)
-        return Response(res)
\ No newline at end of file
+        return Response(res)
+    
+    @action(detail=False, methods=['get'])
+    def mvp_top_three(self, request):
+        all_match_list = self.get_queryset()
+        first_match = all_match_list[0]
+        res= get_match_mvp_data(first_match.id)
+        return Response(res)
+    
+    @action(detail=False, methods=['get'])
+    def teamList(self, request):
+        all_match_list = self.get_queryset()
+        first_match = all_match_list[0]
+        res= get_match_team_list_data(first_match.id)
+        return Response(res)
+    
```

Comment:
Both of the above 2 functions mvp_top_three and teamList are incomplete ..... why are you looking for only the first match?..... also teamList function is wrong by convention, use pascal case

Reply:
Sort it out and make necessary changes. Query part giving problem isn't an excuse

Reply:
Solved